Testscripts/Windows/AddHardDisk.ps1

```diff
@@ -180,12 +180,14 @@ function New-HardDrive
             Write-LogInfo "New-HardDrive was passed a bad SCSI Controller ID: $ControllerID"
             return $False
         }
-
         # Create the SCSI controller if needed
-        $sts = New-SCSIController $vmName $server $controllerID
-        if (-not $sts) {
-            Write-LogErr "Unable to create SCSI controller $controllerID"
-            return $False
+        $controlerObj = Get-VMScsiController -VMName $vmName -ControllerNumber $controllerID -ComputerName $server
```

Comment:
I am not sure if this is right in Azure. There is one Azure test calling this.

Reply:
Will run it in Azure, thank you

Reply:
Notice that PERF-STORAGE-4k-IO call this file, in fact, this Get-VMScsiController is also called in the New-SCSIController, so it should be ok add this call here.

I'm curious that it can be run in Azure, since it calls new-vhd, and all use hyper-v command. I just run Azure, need to wait for test result, will update to you tomorrow.

Reply:
@juhlee-microsoft PERF-STORAGE-4k-IO uses <setupType> oneVM12Disk<setupType> to add disk, no call the file AddHardDisk.ps1, I think this change does not impact Azure, it is safe to merge.

Thank you so much.